content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,319 +10,338 @@ further_reading:
-      exporters: [datadog]
-      # ...
-    ```
-<div class="alert alert-warning">If you get the error <code>unknown processors type "k8sattributes" for k8sattributes</code>, upgrade to the latest OpenTelemetry Collector (v0.37.0 or greater).</div>
+```yaml
+# ...
+k8sattributes:
+  passthrough: true
+# ...
+```
 
-#### Example Kubernetes OpenTelemetry application configuration
+To read more about the `passthrough` option, check [its documentation][23].
 
-In addition to the OpenTelemetry Collector configuration, ensure that OpenTelemetry SDKs that are installed in an application transmit telemetry data to the collector, by configuring the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` with the host IP. Use the downward API to pull the host IP, and set it as an environment variable, which is then interpolated when setting the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable:
+Lastly, make sure that the gateway Collector's configuration uses the Datadog Exporter settings that have been removed from the agents.
 
-```
-apiVersion: apps/v1
-kind: Deployment
-...
-spec:
-  containers:
-  - name: <CONTAINER_NAME>
-    image: <CONTAINER_IMAGE>/<TAG>
-    env:
-      - name: HOST_IP
-        valueFrom:
-          fieldRef:
-            fieldPath: status.hostIP
-        # This is picked up by the opentelemetry sdks
-      - name: OTEL_EXPORTER_OTLP_ENDPOINT
-        value: "http://$(HOST_IP):4317"
+### Alongside the Datadog Agent
+
+To use the OpenTelemetry Collector alongside the Datadog Agent, you will need an additional DaemonSet to ensure that the Datadog Agent runs on each host alongside the previously set up [OpenTelmetry Collector DaemonSet](#daemonset-deployment). To learn how to deploy the Datadog Agent in Kubernetes, you can [visit our help pages][24].
+
+Next, we'll have to enable OTLP ingestion in the Datadog Agent. In order to learn how to do that, go to [this page][25].
+
+Now that the Datadog Agent is ready to receive OTLP traces and metrics, we'll have to change our [OpenTelemetry Collector Daemonset](#daemonset-deployment) to use the [OTLP exporter][19] instead of the Datadog Exporter. To do so, add it to [your config map][26]:
+
+```yaml
+# ...
+exporters:
+  otlp:
+    endpoint: "${HOST_IP}:4317"
+# ...
 ```
 
-To see more information and additional examples of how you might configure your collector, see [the OpenTelemetry Collector configuration documentation][4].
+Given this change, we should make sure that the `HOST_IP` environment variable is provided [in the DaemonSet][27]:
 
-### Running alongside the Datadog Agent
+```yaml
+# ...
+        env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+# ...
+```
 
-If running the OpenTelemetry Collector on a host with an existing Datadog Agent, replace the Datadog exporter with an OTLP exporter pointing to the Datadog Agent:
+Finally, make sure that the [service pipelines][21] are using it:
```

Comment:
```suggestion
5. Make sure that the [service pipelines][21] are using OTLP:
```
I wasn't sure what "it" was referring to. If it's not OTLP, update this to say what it is.